### PR TITLE
BUG solve issue by calling is_accessible_code with submission ID

### LINKED ISSUE #406: Issue with the "join event" button
It appears that under some version of chrome and firefox, the "join event" does not work.

### PR DESCRIPTION
**Pull request by glemaitre**
closes #406

### CODE REVIEW

**Comment by glemaitre** [comment]
ping @agramfort @lucyleeow @maikia We need to make a new 0.4 bug fix release if possible and deploy on the server. Just need to port this PR in 0.4.X branch then

**Comment by glemaitre** [comment]
I might try to do that tonight.

**Comment by glemaitre** [comment]
Nice failure, the test will fail every 4 years :)

File: doc/whats_new/v0.4.rst [hunk 1]
```diff
@@ -19,5 +19,9 @@ Changelog
   via the frontend.
   :pr:`397` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- Bug: fix a bug for which a user could not check is own code in
```
Comment (lucyleeow) [inline doc/whats_new/v0.4.rst]:
```suggestion
- Bug: fix a bug for which a user could not check their own code in
```
'their' is gender neutral option (I know in French general is always masculine)

Comment (lucyleeow) [reply doc/whats_new/v0.4.rst]: Also I like that you drop the 'h' even when writing! :laughing:

Comment (glemaitre) [reply doc/whats_new/v0.4.rst]: :) no need for it

**Pull request merged**